partition_snapshot_reader.hh

```diff
@@ -449,6 +449,7 @@ private:
 
     void do_fill_buffer() {
         while (!is_end_of_stream() && !is_buffer_full()) {
+            check_aborted();
```

Comment:
Note, the abort checks are mostly important on things that depend on I/O. This doesn't hurt and can help in some scenarios, so we may as well keep it.